includes/fbbot/class-omise-fbbot-payment-handler.php

```diff
@@ -0,0 +1,253 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+if (  class_exists( 'Omise_FBBot_Payment_Handler') ) {
+  return;
+}
+
+class Omise_FBBot_Payment_Handler {		
```

Comment:
@igroomgrim I think this one can be moved into `includes/gateway/class-omise-payment-fbbot` and maybe you can implement something like below?

```php
<?php

class Omise_Payment_Fbbot extends Omise_Payment {
    /**
     * Just example
     */
    public function process_payment_alipay() {
        $available_gateways = WC()->payment_gateways->get_available_payment_gateways();

        if ( isset( $available_gateways['omise_alipay'] ) ) {
            // Process payment.
        }
    }

    public function process_payment_creditcard() {
        $available_gateways = WC()->payment_gateways->get_available_payment_gateways();

        if ( isset( $available_gateways['omise'] ) ) {
            // Process payment.
        }
    }

    /**
     * Just example
     */
    public function process_payment_internetbanking() {
        $available_gateways = WC()->payment_gateways->get_available_payment_gateways();

        if ( isset( $available_gateways['omise_alipay'] ) ) {
            // Process payment.
        }       
    }
}
```

Then we can get rid of 

```php
$settings = get_option( 'woocommerce_omise_settings', null );

if ( is_null($settings) || ! is_array( $settings ) ) {
  return;
}

$this->test_mode = isset( $settings['sandbox'] ) && $settings['sandbox'] == 'yes';
$this->private_key = $this->test_mode ? $settings['test_private_key'] : $settings['live_private_key'];
$this->public_key  = $this->test_mode ? $settings['test_public_key'] : $settings['live_public_key'];

if ( empty( $this->private_key ) || empty( $this->public_key ) ) {
  return;
}

// Set omise key
$this->set_oms_key();
```

Because `Omise_Payment` already provided these.